xml/storage_multipath.xml

```diff
@@ -179,13 +109,7 @@
     Storage arrays from the following vendors have been tested with
     &productname;:
    </para>
-   <simplelist>
-    <member>EMC</member>
-    <member>Hitachi</member>
-    <member>Hewlett-Packard/Compaq</member>
-    <member>IBM</member>
-    <member>NetApp</member>
-    <member>SGI</member>
+   <simplelist><member>EMC</member><member>Hitachi</member><member>Hewlett-Packard/Compaq</member><member>IBM</member><member>NetApp</member><member>SGI</member>
```

Comment:
again, this seems wrong, should have line breaks between 'member' tags

Reply:
I blame the daps-xmlformat tool for this. #grump

Reply:
Fixed.